### ISSUE TITLE
Question: Should we new able to see watched episodes? 

### ISSUE THREAD

**Issue by skaforey** [asciidisco/plugin.video.netflix#568]
Running the latest version from the repo on latest LibreElec official build which is based on v18rc1. I was wondering of we should be able to see which episodes we've watched? Also is resume supported?

**Comment by roopeh** [comment]
offtopic: the latest Milhouse's build is rc3...

**Comment by skaforey** [comment]
I have that option on but it doesn't seem to know what I've watched on other devices. Out is supposed to?

**Comment by jakermx** [comment]
nope, each device keep a tracking info by itself, because its esn, look at your streaming activity in netflix web... do you see you history on kodi addon?

**Comment by futurefutsys** [comment]
i had a netflix support issue and the tracking is netflix on layer7 as web api the following occurs:by start they use a ptr to aws there they set 2 ws: (websockets) open on the outgoing ip, in the first they create a proxy by sending a push saying src=(this is the tcp port source from the connect so always random, but this source port makes in layer3 a 3 tcp connection on akamais stack, the web player pushes all kind off nonsens due they make synch callbacks, which is useless due the content is not parsed in the DOM, but akamai does in this proxy an inline dns call, for soho users they have no clue so all goes well, for pro's like us we have on our inline dns no externall intrussion as that is an attack vector, issue the ip makes an dns inq call and unbound does a chase due the ip is a proxy no ns master can sign so at the end the syn cache get's overflooded and the 3 tcp streams connects get broken, for the userspace the session says continue watching , so fetching inside kodi is not kodi bug but dns from akamai is ASN called Microsoft and they have in the os 3 proxys running so the app is all for the shine, the real deal is the way microsoft uses akamai for real data encapsulation and here is the unbound chase and netflix stops streaming , almost one second with 1738 recursion replies sent in an web this kills the state off the 3 proxys and the DOM is overcached so 2 problems DOM does garbage collection and the 3 states tcp in akamai get's killed , the bug is the way netflix operates with aws, 3 proxys and akamai, , that's why they opt for mobile devices due they have no security and other tv api's uses mpeg data frames in the vrowser netflix uses a RTSP as protocol so no application layer, it's a way to solve in 3 different cases and the nr1 is the state thet link on the source port keeping alive since this is in layer7 cached in the local data off the session once this is re-established the complete runtime needs to be set with the cache holding the last sequense, a vm running an android os and bridging that in an isoleted stack with full dns accesss and a virtual ip same as public only the real ip is a lan fools the proxy and kodi will stream simply like a honeypot

Dec 12 05:18:54 | unbound | 69869:2 | info: [25%]=0.0730813 median[50%]=0.499252 [75%]=1.01807
-- | -- | -- | --
Dec 12 05:18:54 | unbound | 69869:2 | info: histogram of recursion processing times
Dec 12 05:18:54 | unbound | 69869:2 | info: average recursion processing time 1.229596 sec
Dec 12 05:18:54 | unbound | 69869:2 | info: mesh_run: end 0 recursion states (0 with reply, 0 detached),  0 waiting replies, 1738 recursion replies sent, 0 replies dropped, 0  states jostled out
Dec 12 05:18:54 | unbound | 69869:2 | debug: query took 0.858193 sec
Dec 12 05:18:54 | unbound | 69869:2 | debug: mesh_run: validator module exit state is module_finished
Dec 12 05:18:54 | unbound | 69869:2 | debug: val handle processing q with state VAL_FINISHED_STATE
Dec 12 05:18:54 | unbound | 69869:2 | info: Validate: message contains bad rrsets
